test/helpers.tsx

```diff
@@ -1,24 +1,102 @@
+import "date-fns";
 import { bigtestGlobals } from "@bigtest/globals";
 import { render as rtlRender } from "@testing-library/react";
-import { ReactElement } from "react";
+import { ComponentProps, ComponentType, ReactElement, useState } from "react";
 import { StylesProvider, jssPreset } from "@material-ui/core/styles";
 import { create } from "jss";
+import { MuiPickersUtilsProvider } from "@material-ui/pickers";
+import DateFnsUtils from "@date-io/date-fns";
 
 export function render(element: ReactElement) {
-  let insertion = bigtestGlobals.document.createComment("mui-jss-insertion");
-  let insertionPoint = bigtestGlobals.document.head.insertBefore(insertion, bigtestGlobals.document.head.firstChild);
-  const jss = create({
-    ...jssPreset(),
-    // Define a custom insertion point that JSS will look for when injecting the styles into the DOM.
-    insertionPoint,
-  });
-
-  return rtlRender(
-    <StylesProvider jss={jss} injectFirst>
-      {element}
-    </StylesProvider>,
-    {
-      container: bigtestGlobals.document.body,
-    }
-  );
+  return {
+    description: "render",
```

Comment:
Is there no other diagnostic information we can add to this step, like the JSX in string form, or even the element name?

Reply:
I can add the `description` argument this function